tests/lib/helper.bash

```diff
@@ -132,6 +133,16 @@ teardown() {
     rm -rf "${TEST_TEMP_DIR}/home/.gnupg/"
 
     temp_del "${TEST_TEMP_DIR}"
+
+    # HOME created by setup
+    temp_del "${TEST_TEMP_HOME}"
+
+    # some tests create their own HOME
+    if [ "${HOME}" != "${TEST_TEMP_HOME}" ]; then
+        if [ "${HOME}" != "${REAL_HOME}" ]; then
+            temp_del "${HOME}"
```

Comment:
This feels dangerous for me.

Is it better to find such tests that defined a own HOME and move the HOME as sub folder to `$TEST_TEMP_HOME` ?

Like
https://github.com/jkroepke/helm-secrets/blob/9f88bc76611304040da99fd378f4db73f2f7a1d1/tests/unit/plugin-install.bats#L9

to `HOME="${TEST_TEMP_HOME}/$(mktemp -d)"`.

Then `temp_del "${TEST_TEMP_HOME}"`  should cover such tests. What did you think?

Reply:
Sure, it would be a lot better. There is only `unit/plugin-install.bats` which is concerned, I will update this PR.